apimonitor/monitor/views.py

```
@@ -0,0 +1,179 @@
+    HealthCheckResultSerializer,
+    AIDigestSerializer,
+    ServiceHealthSummarySerializer
+)
+
+
+class ServiceViewSet(viewsets.ModelViewSet):
+    """ViewSet for Service CRUD operations."""
+    
+    queryset = Service.objects.all()
+    serializer_class = ServiceSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)